code/index.html

```diff
@@ -1,30 +1,126 @@
+            <div class="scroll-down"> </div>
+        </a>
+
     </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+<!-- second section with the forms/radio-buttons/checkboxes/selector and subscribe button -->
+    <section class="subscribe" id="sec-2">
+        <div>
+            <div class="bg">
+                <form action="http://httpbin.org/anything" method="post" class="forms">
+                    <input class="formq" type="text" id="name" name="fullname" placeholder="Name">
+                    <input class="formq" type="date" id="birthday" name="birthday">
+                    <input class="formq" type="text" id="email" name="email" placeholder="Email">
+                    <input class="formq" type="text" id="phonenumber" placeholder="Phone Number">
+                    <input class="formq" type="password" id="password " placeholder="Password ">
+                    <div class="radiodiv">
+                        <div class="radiobg">
+                    <label class="container">Red Package
+                        <input type="radio" checked="checked" name="radio">
+                        <span class="checkmark"></span>
+                      </label>
+                    </div>
+                    <div class="radiobg">
+                      <label class="container">Blue Package
+                        <input type="radio" name="radio">
+                        <span class="checkmark"></span>
+                      </label>
+                    </div>
+                    <div class="radiobg">
+                      <label class="container">Green Package
+                        <input type="radio" name="radio">
+                        <span class="checkmark"></span>
+                      </label>
+                    </div>
+                </div>
+                
+                    <select name="gender ">
+                      <option value="Choose ">Choose</option>
+                      <option value="Male ">Male</option>
+                      <option value="Female ">Female</option>
+                      <option value="Intersex/non-binary ">Intersex/non-binary</option>
+                      <option value="Prefer not to respond ">Prefer not to respond</option>
+                   </select>
+                    
+                   <div class="terms">
+                       <input class="termscheck" type="checkbox" id="checkbox">
+                       <label for="checkbox">I agree to these <a href="#">Terms and Condition</a></label>
+                   </div>
+                   <button class="sub-button">Subscribe</button>
+                
+                </form>
+             </div>
+
+            <!-- second animation button that takes you to section 3 -->
+        </div>
+        <a href="#sec-3">
+            <div class="scroll-down halfling " id="scroll-two "></div>
+        </a>
+    </section>
+
+<!-- the FAQ section -->
+    <section class="accordion" id="sec-3">
```

Comment:
The accordion looks good! I the + and - sign is a great touch, and something i would have added too if i had the time! good job.